task2b.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import time
+
+start2_time = time.time()
+
+x = np.random.rand(10)
+y = np.random.rand(10)
+
+sum = 0
+i=0
+for a in x:
+	sum = sum + (x[i]*y[i])
+	i=i+1
```

Comment:
You should write the loop something like this:
for i in range(len(x)):
    sum = sum + x[i]*y[i]